pkg/storaged/nfs-details.jsx

```diff
@@ -17,8 +17,6 @@
  * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
  */
 
-import "polyfills";
```

Comment:
To clarify: It's inconsistent to import it in a component; I moved that to devices.jsx, which is the top-level entry for the Storage page.